var/spack/repos/builtin/packages/slate/package.py

```diff
@@ -47,16 +41,15 @@ def install(self, spec, prefix):
         f_cuda = "1" if spec.variants['cuda'].value else "0"
         f_mpi = "1" if spec.variants['mpi'].value else "0"
         f_openmp = "1" if spec.variants['openmp'].value else "0"
-        f_mkl = "1" if spec.variants['mkl'].value else "0"
 
         compiler = 'mpicxx' if spec.variants['mpi'].value else ''
 
         make('mpi=' + f_mpi, 'mkl=1', 'cuda=' + f_cuda, 'openmp=' + f_openmp,
```

Comment:
Hi @G-Ragghianti, I think the original intent was that the `mkl` variant would always be true for now. Was the double specification of `mkl=` causing an issue with the build? You could remove the hardcoded `mkl=1` instance and keep the `mkl= f_mkl` instance, which I think would be equivalent and allow this package to continue including hints about future general cblas/scalapack support.

Reply:
The problem was that the "mkl" variant was commented out, causing the "variants['mkl']" key lookup to fail on line 50.  Therefore, line 50 and part of line 55 would have to be commented out as well (all references to 'mkl').  Rather than do this, I just remove all references completely.